### PR TITLE
Feature/static config

### PR DESCRIPTION
**Pull request by romeovs**
Adds a helper package for defining static configurations.

### CODE REVIEW

File: configuration/configuration.go [hunk 1]
```diff
@@ -0,0 +1,264 @@
+package configuration
```
Comment (htdvisser) [inline configuration/configuration.go]: Missing copyright header

File: configuration/configuration_test.go [hunk 2]
```diff
@@ -0,0 +1,262 @@
+package configuration
```
Comment (htdvisser) [inline configuration/configuration_test.go]: Missing copyright header

File: configuration/example_test.go [hunk 3]
```diff
@@ -0,0 +1,69 @@
+package configuration
```
Comment (htdvisser) [inline configuration/example_test.go]: - Missing copyright header
- Use `configuration_test` package

File: configuration/errors.go [hunk 4]
```diff
@@ -0,0 +1,33 @@
+package configuration
```
Comment (htdvisser) [inline configuration/errors.go]: Missing copyright header

File: configuration/configuration.go [hunk 5]
```diff
@@ -0,0 +1,264 @@
...
+
+	"github.com/spf13/pflag"
+	"github.com/spf13/viper"
+)
+
+// Definition is a configuration definition
+type Definition struct {
+	defaults interface{}
+	pf       *pflag.FlagSet
+	viper    *viper.Viper
```
Comment (htdvisser) [inline configuration/configuration.go]: Just embed viper

**Comment by romeovs** [comment]
@htdvisser Headers are there!

**Pull request closed without merging**